analysis/topEFT/get_yield_json.py

```diff
@@ -0,0 +1,45 @@
+import argparse
+import json
+import datetime
+from topcoffea.modules.YieldTools import YieldTools
+import topcoffea.modules.MakeLatexTable as mlt
+
+# This script takes a pkl file, finds the yields in the analysis categories, saves the yields to a json
+# Example usage:
+# python get_yield_json.py histos/plotsTopEFT.pkl.gz
+
+def main():
+
+    yt = YieldTools()
+
+    timestamp_tag = datetime.datetime.now().strftime('%Y%m%d_%H%M')
+
+    # Set up the command line parser
+    parser = argparse.ArgumentParser()
+    parser.add_argument("pkl_file_path", default="hists/plotsTopEFT.pkl.gz", help = "The path to the pkl file")
```

Comment:
Since `pkl_file_path` is a required argument, providing a default value doesn't make sense.  Omitting this argument raises an error, so the default value will never get used.  Did you mean to allow people to run the script without specifying the pkl file?  (It's possible, but not like this.)

Reply:
Okay, that makes sense, thanks. The pkl file path should now be optional, with a default of `hists/plotsTopEFT.pkl.gz`. The commit is [here](https://github.com/TopEFT/topcoffea/pull/49/commits/78d5d8595164fc2419e9f79507f9d2e3eedcf053#diff-dbb4eb7234b717b49581da1da6136864f3f8865de29cb6e21ad1d81eb0bc3e88).